dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -268,7 +269,13 @@ case class BinaryOutcomeDLCWithSelf(
     */
   def executeDLC(
       oracleSigF: Future[SchnorrDigitalSignature],
-      local: Boolean): Future[(Transaction, BitcoinUTXOSpendingInfo)] = {
+      local: Boolean): Future[
+    (
```

Comment:
Can we a create a new type for this with documentation rather than a massive nested tuple?